src/test/java/org/openmrs/performance/http/DoctorHttpService.java

```diff
@@ -148,11 +152,78 @@ public HttpRequestActionBuilder getDrugOrders(String patientUuid) {
 						"&status=any&orderType=" + DRUG_ORDER +
 						"&v=" + customRepresentation);
 	}
+
+	public HttpRequestActionBuilder searchPatient(String searchQuery) {
+		String customRepresentation = """
+				custom:(patientId,uuid,identifiers,display,patientIdentifier:(uuid,identifier),person:(gender,age,birthdate,birthdateEstimated,personName,addresses,display,dead,deathDate),attributes:(value,attributeType:(uuid,display)))
+				""";
+		return http("Get Orders")
+				.get("/openmrs/ws/rest/v1/patient" +
+						"?q=" + searchQuery +
+						"&v=" + customRepresentation +
+						"&includeDead=" + true +
+						"&limit=" + 10);
+	}
+		
+	
 	
 	public HttpRequestActionBuilder getAllergies(String patientUuid) {
 		return http("Get Allergies of Patient")
 				.get("/openmrs/ws/fhir2/R4/AllergyIntolerance?patient=" + patientUuid + "&_summary=data");
 	}
+
+	public HttpRequestActionBuilder GetDrugAllergens(String drugAllergenUuid) {
+		return http("Get Drug Allergens")
+				.get("/openmrs/ws/rest/v1/concept/" + drugAllergenUuid + "?v=full");
+	}
+
+	public HttpRequestActionBuilder GetEnvironmentAllergens(String environmentalAllergenUuid) {
+		return http("Get Environment Allergens")
+				.get("/openmrs/ws/rest/v1/concept/" + environmentalAllergenUuid + "?v=full");
+	}
+
+	public HttpRequestActionBuilder GetFoodAllergens(String foodAllergenUuid) {
+		return http("Get Food Allergens")
+				.get("/openmrs/ws/rest/v1/concept/" + foodAllergenUuid + "?v=full");
+	}
+
+	public HttpRequestActionBuilder GetAllergicReactions(String allergyReactionUuid) {
+		return http("Get Allergic Reactions")
+				.get("/openmrs/ws/rest/v1/concept/" + allergyReactionUuid + "?v=full");
+	}
+
+	public HttpRequestActionBuilder saveAllergies(String patientUuid) {
+		Map<String, Object> payload = new HashMap<>();
+
+		Map<String,String> codedAllergen = new HashMap<>();
+		codedAllergen.put("uuid", "71617AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA");
```

Comment:
Let's make the UUID a constant and move it to the [Constants.java](https://github.com/openmrs/openmrs-contrib-performance-test/blob/bcab59e385b1753b52a6e93759b5b0d4e3a832d4/src/test/java/org/openmrs/performance/Constants.java) file. That way it’ll be super clear what this UUID is for and we can easily reuse it in other parts of the code.